config/config.go

```diff
@@ -224,6 +229,7 @@ type (
 		ID              uint32           `yaml:"id"`
 		Address         string           `yaml:"address"`
 		ProducerPrivKey string           `yaml:"producerPrivKey"`
+		SignatureScheme []string         `yaml:"signatureScheme"`
```

Comment:
I thought we start with one valid scheme

Reply:
if the chain wants only 1 valid scheme, then put only 1 in the config

the default is []string{SigP256k1}, which is the current scheme supported